en/sim_gazebo_gz/README.md

```diff
@@ -163,8 +172,10 @@ where `ARGS` is a list of environment variables including:
   Sets the simulator, which for Gz must be `gz`.
   - This value should be [set for the selected airframe](#adding-new-worlds-and-models), in which case it does not need to be set as an argument.
 
-The PX4 Gazebo worlds and and models databases [can be found on Github here](https://github.com/PX4/PX4-Autopilot/tree/main/Tools/simulation/gz).
-They are added to the Gazebo search `PATH` by [gz_env.sh.in](https://github.com/PX4/PX4-Autopilot/blob/main/src/modules/simulation/gz_bridge/gz_env.sh.in) during the simulation startup phase.
+- `PX4_GZ_SIM`:
+  Lets PX4 know that it should launch an instance of Gazebo.
+
+The PX4 Gazebo worlds and and models databases [can be found on Github here](https://github.com/PX4/PX4-gazebo-models).
 
 :::note
```

Comment:
Remove this please, `gz_env_sh.in` has ben removed in PX4-Autopilot

Reply:
Wrong line matching, I meant

:::note
`gz_env.sh.in` is compiled and made available in `$PX4_DIR/build/px4_sitl_default/rootfs/gz_env.sh`
:::